platforms/cpuinfo.go

```diff
@@ -98,7 +101,11 @@ func getCPUVariant() string {
 	variant, err := getCPUInfo("Cpu architecture")
 	if err != nil {
 		log.L.WithError(err).Error("failure getting variant")
```

Comment:
Looks like the existing code also treats `NotImplemented` the same as "an actual error" occurred; we should probably have a `errdefs.IsNotImplemented()` check here as well to prevent bogus errors from being logged.